packages/bemuse-tools/src/indexer.js

```diff
@@ -1,18 +1,17 @@
-import Promise from 'bluebird'
-import co from 'co'
 import fs from 'fs'
 import _ from 'lodash'
 import chalk from 'chalk'
 import json from 'format-json'
 import yaml from 'js-yaml'
+import util from 'util'
 
 import { getSongInfo } from 'bemuse-indexer'
 import { join, dirname, basename } from 'path'
 
-let fileStat = Promise.promisify(fs.stat, fs)
-let readFile = Promise.promisify(fs.readFile, fs)
-let writeFile = Promise.promisify(fs.writeFile, fs)
-let glob = Promise.promisify(require('glob'))
+let fileStat = util.promisify(fs.stat, fs)
+let readFile = util.promisify(fs.readFile, fs)
+let writeFile = util.promisify(fs.writeFile, fs)
```

Comment:
Thanks for also removing Bluebird. However, `util.promisify` only takes one argument. Can you help fix this?

Reply:
Should I update all references to Bluebird to use native Promise and the promisify function from util like I did here?

Reply:
I agree... I would like to move away from using Bluebird instance methods. I'd prefer it in a separate PR though to make it easy for me to review and test...

Do you plan to update the whole project?

Reply:
Sure, why not. Let me create a new PR without Bluebird.